CanalTP/Chaos
# Create draft table for drafts/disruptions API

# Description

This PR adds draft table into the db in order to implement  drafts/disruptions API

## Issue (optional)

Issue link:BOT-3343

## Pull Request type (optional)

This is a new feature?

## How to test

Here are the following steps to test this pull request:

- run db migration `honcho run ./manage.py db upgrade`
- check that draft table was created into the db
